explainer.md

```diff
@@ -94,18 +99,50 @@ There are situations where the design of GPC, by intent, matches specific legal
 
 ### 4.1 GPC in the US
 
-The [CCPA Regulations](https://oag.ca.gov/privacy/ccpa/regs) provide specific language supporting consumer’s use of a global privacy control. GPC was created with the intent to supply users with the ability to exercise CCPA and related opt out requests. GPC compliance is [enforced by the Office of the California Attorney General](https://oag.ca.gov/news/press-releases/attorney-general-bonta-announces-settlement-sephora-part-ongoing-enforcement) and the California Privacy Protection Agency.
+Since 2018, at least nineteen states have passed comprehensive state privacy laws that include, among other rights, the right to opt out of the sale or sharing of their personal information and/or the right to opt out of cross-context targeted advertising. Many of these laws explicitly state that consumers may exercise these rights through a universal signal, including a signal sent through a browser or operating system. At least two states laws — California and Colorado — state that receipt of a Global Privacy Control signal should be interpreted as a legally binding opt-out right in that state.
```

Comment:
At least two states laws — California and Colorado — -> At least two laws — those of California and Colorado —

should be interpreted as a legally binding opt-out right in that state. -> is to be interpreted as a legally binding exercise of the opt-out right in that state.